src/parser/jobs/blm/modules/Sharpcast.js

```diff
@@ -0,0 +1,136 @@
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import React from 'react'
+import {Item} from 'parser/core/modules/Timeline'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Trans, Plural} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+
+const SHARPCAST_DURATION_MILLIS = STATUSES.SHARPCAST.duration * 1000
+
+const SHARPCAST_CONSUMER_IDS = [
+	ACTIONS.FIRE_I.id,
+	ACTIONS.THUNDER_III.id,
+	ACTIONS.THUNDER_IV.id,
+	ACTIONS.SCATHE.id,
```

Comment:
Do you think it would be worth tracking how many Sharpcasts are consumed by each individual action? I'm no BLM main, but I'm wondering if it would be worth adding a suggestion that yells at people for wasting it on Scathe (unless that's not a waste anymore).

Reply:
It's definitely a waste to use Sharpcast on it, but there is something to be said for not yelling at them if Sharpcast comes off CD, gets popped, then gets consumed on a Scathe as the boss goes invuln, and the CD is back up by the time the invuln period is over. They would've wasted the Sharpcast in that case, so I don't want to ding them for using it on a normally-suboptimal action that was actually the optimal one in the niche case.